python/paddle/fluid/framework.py

```diff
@@ -554,7 +561,12 @@ def find_name(var_list, name):
                         (out_proto.name, len(out_args)))
                 out_arg_names = []
                 for arg in out_args:
-                    out_arg_names.append(arg.name)
+                    if isinstance(arg.name, six.string_types):
+                        out_arg_names.append(arg.name)
+                    elif isinstance(arg.name, six.binary_type):
+                        out_arg_names.append(arg.name.decode())
+                    else:
```

Comment:
when will else happen?

Reply:
If we meet `bytes`, we should decode it rather than ignoring it, this is a bug, I fixed it here

Reply:
six.binary_type covers py3 bytes?

Reply:
https://pythonhosted.org/six/

Reply:
Yes, here is the official description 
> six.binary_type
Type for representing binary data. This is str in Python 2 and bytes in Python 3.